docs/src/guide/installation.md

```diff
@@ -250,6 +251,79 @@ Download and run the [Installer][doxypress_binaries].
 Please use the free and open-source [7-Zip file archiver][7_zip] to extract the
 files in the next steps.
 
+## Windows with WSL2
+
+The [Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/about#what-is-wsl-2) allows you to run a Linux distribution in parallel to Windows. In order to use this Linux for developing you need to [bridge the USB hub to which the device is attached](https://docs.microsoft.com/en-us/windows/wsl/connect-usb).
+
+- Install Ubuntu-22.04 LTS as WSL2 distribution
+  - See the [general WSL doc](https://docs.microsoft.com/en-us/windows/wsl/install), but
+  - as of this writing `wsl --list --online` will not list Ubuntu 22.04.1 LTS, but it is [available in the Microsoft Store](https://www.microsoft.com/store/productId/9PN20MSR04DW). Install it from there via mouse clicks
+- Install [Visual Studio Code](https://code.visualstudio.com/) and the [remote extensions for WSL](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-vscode) and make sure you can log in to your linux distribution
+- Physically attach the microcontroller board to a USB port
```

Comment:
I begin to understand why I have hard times digging through modm docs ... if you have to click into many different places to collect all relevant information just to have things up and running it's no fund and let people turn away for no good reason. 

IMHO it **is** a relevant information because people might not guess that this has to be done before issuing the other commands and fail on the bridge.